### ISSUE TITLE
Assertion error when restarting and running.

### ISSUE THREAD

**Issue by docktermj** [Senzing/senzing-with-chatgpt-notebook#5]
When running `Kernel` > `Restart & Run All`, In Test's 4th cell, this error occurs.

The cell:
```python
function_message = function_call(messages[-1])
messages.append(function_message)

messages = chat(messages)
```
The result:
```console
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
Cell In[13], line 1
----> 1 function_message = function_call(messages[-1])
      2 messages.append(function_message)
      4 messages = chat(messages)

Cell In[10], line 2, in function_call(msg)
      1 def function_call(msg):
----> 2     assert 'assistant' == msg['role']
      3     assert not msg['function_call'] is None
      5     func_name = msg['function_call']['name']

AssertionError: 
```

**Comment by docktermj** [comment]
Also seeing this error:

The cell:
```python
messages = [dict(role='user', content='Show me examples of matches')]
messages = chat(messages)
```
Error:
```json
{'error': {'code': 'model_not_found',
           'message': 'The model: `gpt-4-0613` does not exist',
           'param': None,
           'type': 'invalid_request_error'}}
```

**Comment by kakugawa** [comment]
I believe it's because you need gpt-4 api access. https://openai.com/waitlist/gpt-4-api

**Comment by docktermj** [comment]
We should add to the README.md a step for "gpt-4 api access" and a step of how to verify that `gpt-4` access is granted.

**Comment by kakugawa** [comment]
That's a good point on verifying that `gpt-4` access. I'll make these changes.

**Comment by kakugawa** [comment]
Fixed in #6
